### PR TITLE
Remove FolderID from service tests

### PR DESCRIPTION
**Pull request by idafurjes**
Removes folderID from most of the tests in the services package.

Ref https://github.com/grafana/grafana/issues/78757

### CODE REVIEW

File: pkg/services/dashboardimport/service/service_test.go [hunk 1]
```diff
@@ -58,7 +58,6 @@ func TestImportDashboardService(t *testing.T) {
 		}
 		folderService := &foldertest.FakeService{
 			ExpectedFolder: &folder.Folder{
-				ID:  5, // nolint:staticcheck
 				UID: "123",
```
Comment (yangkb09) [inline pkg/services/dashboardimport/service/service_test.go:61]: should we add an `OrgID` here?

File: pkg/services/dashboards/accesscontrol_test.go [hunk 2]
```diff
@@ -27,7 +27,7 @@ func TestNewFolderNameScopeResolver(t *testing.T) {
 		orgId := rand.Int63()
 		title := "Very complex :title with: and /" + util.GenerateShortUID()
 		// nolint:staticcheck
-		db := &folder.Folder{Title: title, ID: rand.Int63(), UID: util.GenerateShortUID()}
+		db := &folder.Folder{Title: title, UID: util.GenerateShortUID()}
```
Comment (yangkb09) [inline pkg/services/dashboards/accesscontrol_test.go:30]: should we add an `OrgID` here?

File: pkg/services/dashboards/database/database_test.go [hunk 3]
```diff
@@ -90,24 +88,6 @@ func TestIntegrationDashboardDataAccess(t *testing.T) {
 		require.False(t, queryResult.IsFolder)
 	})
 
-	t.Run("Should be able to get dashboard by title and folderID", func(t *testing.T) {
```
Comment (yangkb09) [inline pkg/services/dashboards/database/database_test.go:93]: genuine Q - how do you know we can delete this test? do we not need it anymore?

File: pkg/services/dashboards/models_test.go [hunk 4]
```diff
@@ -66,12 +66,10 @@ func TestSaveDashboardCommand_GetDashboardModel(t *testing.T) {
 		json := simplejson.New()
 		json.Set("title", "test dash")
 
-		// nolint:staticcheck
-		cmd := &SaveDashboardCommand{Dashboard: json, FolderID: 1, FolderUID: "1"}
+		cmd := &SaveDashboardCommand{Dashboard: json, FolderUID: "1"}
```
Comment (yangkb09) [inline pkg/services/dashboards/models_test.go:69]: should we provide `OrgID` here?

File: pkg/services/dashboards/database/database_test.go (hunk 3 above)
Comment (idafurjes) [reply pkg/services/dashboards/database/database_test.go:93]: it is testing get dashboard by title and `folderID`, we are deprecating folderIDs, we are removing all the folderIDs from the tests, if the test case is testing folderID specific behavior we can remove it

File: pkg/services/dashboards/accesscontrol_test.go (hunk 2 above)
Comment (idafurjes) [reply pkg/services/dashboards/accesscontrol_test.go:30]: why do you think we should add an orgID there?

File: pkg/services/dashboards/accesscontrol_test.go [hunk 5]
```diff
@@ -27,7 +27,7 @@ func TestNewFolderNameScopeResolver(t *testing.T) {
 		orgId := rand.Int63()
 		title := "Very complex :title with: and /" + util.GenerateShortUID()
 		// nolint:staticcheck
```
Comment (zserge) [inline pkg/services/dashboards/accesscontrol_test.go:29]: We should be careful about leftover nolint directives - they would become hard to notice and to remove later

**Review by zserge — APPROVED** [review]

File: pkg/services/dashboards/accesscontrol_test.go (hunk 5 above)
Comment (idafurjes) [reply pkg/services/dashboards/accesscontrol_test.go:29]: not really I am searching for this lines in pkgs, that's my method for removing them, but will remove this one

File: pkg/services/dashboards/models.go [hunk 6]
```diff
@@ -301,8 +301,9 @@ type GetDashboardQuery struct {
 	UID   string
 	Title *string
 	// Deprecated: use FolderUID instead
-	FolderID *int64
-	OrgID    int64
+	FolderID  *int64
+	FolderUID string
```
[the remark this answers is not in the document]
Comment (idafurjes) [reply pkg/services/dashboards/models.go:305]: I am not touching code, only tests 🤞

File: pkg/services/dashboards/database/database_test.go (hunk 3 above)
Comment (papagian) [reply pkg/services/dashboards/database/database_test.go:93]: I think we need to make sure that we introduce a similar test using `folderUID` instead

File: pkg/services/dashboards/database/database_test.go [hunk 7]
```diff
@@ -136,18 +99,6 @@ func TestIntegrationDashboardDataAccess(t *testing.T) {
 		require.ErrorIs(t, err, dashboards.ErrDashboardIdentifierNotSet)
 	})
 
-	t.Run("Folder=0 should not be able to get a dashboard in a folder", func(t *testing.T) {
```
Comment (papagian) [inline pkg/services/dashboards/database/database_test.go:139]: same as above: better change it to use the `FolderUID` instead of removing it

**Pull request closed without merging**